cube_convert.py

```diff
@@ -200,10 +204,45 @@
 # Loop through each cut file and assign the channel number, editing the simple
 # metadata (automatically distinguish between a 3-element array or single
 # sensor)
-t_min, t_max = np.inf, -np.inf  # Initialize time bounds
+
+mdayhour = []
 for file in cut_file_list:
+    if file[-7:-6] == '.':
```

Comment:
Can add a comment here explaining the significance of `file[-7:-6] == '.'` ?

Reply:
Yes I can. If there is a gap within one minute the mseedcut command produces two files with the same name but the second file will have a ".1" inserted before the ".pri0" (e.g. c0af0180628020000.pri0 and c0af0180628020000.1.pri0). The for loop and variable names `mdayhour` is a list of strings of the month, day and hour of each file. This is later used to determine if there are multiple files for each hour.